tests/connect/standalone-ssl-verify_cert-error-001.phpt

```diff
@@ -28,5 +28,5 @@ Deprecated: MongoDB\Driver\Manager::__construct(): The "allow_invalid_hostname"
 
 Deprecated: MongoDB\Driver\Manager::__construct(): The "weak_cert_validation" driver option is deprecated. Please use the "tlsAllowInvalidCertificates" URI option instead.%s
 OK: Got MongoDB\Driver\Exception\ConnectionTimeoutException thrown from executeCommand
-No suitable servers found (`serverSelectionTryOnce` set): [%s calling ismaster on '%s:%d']
+No suitable servers found (`serverSelectionTryOnce` set): [%s calling %s on '%s:%d']
```

Comment:
AFAIK, this error message comes from `mongoc-topology-scanner.c`. The template is always going to refer to either `ismaster` or `hello`, so I assume it was preferable to just use `%s` here instead of a regex pattern like `(ismaster|hello)`, correct?

Reply:
It was the easiest, but `(ismaster|hello)` is better.